x/stake/handler.go

```diff
@@ -37,6 +33,46 @@ func NewHandler(k keeper.Keeper) sdk.Handler {
 
 // Called every block, process inflation, update validator set
 func EndBlocker(ctx sdk.Context, k keeper.Keeper) (ValidatorUpdates []abci.Validator) {
+	cdc := types.MsgCdc
+
+	endBlockerTags := sdk.EmptyTags()
+
+	unbondingTimesliceIterator := k.UnbondingQueueIterator(ctx, ctx.BlockHeader().Time)
+	for ; unbondingTimesliceIterator.Valid(); unbondingTimesliceIterator.Next() {
```

Comment:
could we add a comment here describing how this iterator actually works? - it's a big confusing just looking at the code.